opennms-base-assembly/src/main/filtered/bin/opennms

```diff
@@ -390,176 +406,187 @@ doStart(){
 			return 1
 		fi
 		sleep $STATUS_WAIT
-		STATUS_ATTEMPTS=`expr $STATUS_ATTEMPTS + 1`
+		STATUS_ATTEMPTS="$((STATUS_ATTEMPTS++))"
```

Comment:
I don't think this works anymore since the assignment is not capturing the incremented value. Just increment the value without re-assigning.
```suggestion
		((STATUS_ATTEMPTS++))
```